### PR TITLE
added Disclaimer Regarding Google Analytics Data Accuracy

### PR DESCRIPTION
**Pull request by progresak**
#### Changelog notes 

Pleease review the voice + formatting of this information.

```
* Added Disclaimer Regarding Google Analytics Data Accuracy
```

#### Check during review

- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

### CODE REVIEW

File: integrations/README.md [hunk 1]
```diff
@@ -3,5 +3,20 @@
...
+> 
+> **Ad Blockers Impact:**
+> - Approximately 33% of internet users utilize ad blockers, which can block Google Analytics and Google Tag Manager (GTM) scripts. See for example this article for a reference: [https://backlinko.com/ad-blockers-users](https://backlinko.com/ad-blockers-users)
+> - Consequently, the data received from these services may not be 100% accurate.
+> - Metrics such as revenue and the number of bookings may appear lower than the actual figures due to ad-blocking behavior.
+> 
+> **Consent and Tracking:**
+> If a guest declines cookies or tracking, their interactions won’t be tracked in Google Analytics.
+> 
+> Remember to address these considerations when interpreting GA data and making business decisions.
```
Comment (zdrazil) [inline integrations/README.md:18]: nitpick: 
```suggestion
> Please remember to address these considerations when interpreting GA data and making business decisions.
```

**Review by zdrazil — APPROVED** [review]
Nice 👍

File: integrations/README.md [hunk 2]
```diff
@@ -3,5 +3,20 @@
 The Booking Engine Widget supports direct integration with [Google Tag Manager](https://marketingplatform.google.com/about/tag-manager/) and [Google Analytics](https://marketingplatform.google.com/about/analytics/).
 The integration with Google Tag Manager is intended only to serve data to Google Analytics.
 
+> ### 🚨 Disclaimer Regarding Google Analytics Data Accuracy:
```
Comment (MikeAdamsMews) [inline integrations/README.md:6]:
```suggestion
> ### 🚨 Disclaimer regarding Google Analytics data accuracy
```

File: integrations/README.md [hunk 3]
```diff
@@ -3,5 +3,20 @@
 The Booking Engine Widget supports direct integration with [Google Tag Manager](https://marketingplatform.google.com/about/tag-manager/) and [Google Analytics](https://marketingplatform.google.com/about/analytics/).
 The integration with Google Tag Manager is intended only to serve data to Google Analytics.
 
+> ### 🚨 Disclaimer Regarding Google Analytics Data Accuracy: 
+> 
+> We understand that some property owners have expressed concerns about discrepancies between the data they see in Google Analytics (GA) and the actual reality. It’s essential to be aware of the following factors:
+> 
+> **Ad Blockers Impact:**
```
Comment (MikeAdamsMews) [inline integrations/README.md:10]:
```suggestion
> **Ad Blockers impact**
```

File: integrations/README.md [hunk 4]
```diff
@@ -3,5 +3,20 @@
...
+> - Approximately 33% of internet users utilize ad blockers, which can block Google Analytics and Google Tag Manager (GTM) scripts. See for example this article for a reference: [https://backlinko.com/ad-blockers-users](https://backlinko.com/ad-blockers-users)
```
Comment (MikeAdamsMews) [inline integrations/README.md:11]:
```suggestion
> - Approximately 33% of internet users utilize ad blockers, which can block Google Analytics and Google Tag Manager (GTM) scripts. See for example this article for a reference: [https://backlinko.com/ad-blockers-users](https://backlinko.com/ad-blockers-users).
```

File: integrations/README.md [hunk 5]
```diff
@@ -3,5 +3,20 @@
...
+> - Consequently, the data received from these services may not be 100% accurate.
+> - Metrics such as revenue and the number of bookings may appear lower than the actual figures due to ad-blocking behavior.
+> 
+> **Consent and Tracking:**
```
Comment (MikeAdamsMews) [inline integrations/README.md:15]:
```suggestion
> **Consent and tracking**
```

File: integrations/README.md [hunk 6]
```diff
@@ -3,5 +3,20 @@
...
+> If a guest declines cookies or tracking, their interactions won’t be tracked in Google Analytics.
```
Comment (MikeAdamsMews) [inline integrations/README.md:16]:
```suggestion
> - If a guest declines cookies or tracking, their interactions won’t be tracked in Google Analytics.
```

File: integrations/README.md [hunk 7]
```diff
@@ -3,5 +3,20 @@
...
 The integration with Google Tag Manager is intended only to serve data to Google Analytics.
 
+> ### 🚨 Disclaimer Regarding Google Analytics Data Accuracy: 
+> 
+> We understand that some property owners have expressed concerns about discrepancies between the data they see in Google Analytics (GA) and the actual reality. It’s essential to be aware of the following factors:
+> 
+> **Ad Blockers Impact:**
+> - Approximately 33% of internet users utilize ad blockers, which can block Google Analytics and Google Tag Manager (GTM) scripts. See for example this article for a reference: [https://backlinko.com/ad-blockers-users](https://backlinko.com/ad-blockers-users)
+> - Consequently, the data received from these services may not be 100% accurate.
+> - Metrics such as revenue and the number of bookings may appear lower than the actual figures due to ad-blocking behavior.
```
Comment (MikeAdamsMews) [inline integrations/README.md:13]:
```suggestion
> - Metrics such as revenue and the number of bookings may appear lower than the actual figures, due to ad-blocking behavior.
```

File: changelog/README.md [hunk 8]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 10th June 2024
+* Integration: Added [Disclaimer Regarding Google Analytics Data Accuracy](../integrations/google-tag-manager.md)
```
Comment (MikeAdamsMews) [inline changelog/README.md:4]:
```suggestion
* Integration: Added [Disclaimer regarding Google Analytics data accuracy](../integrations/google-tag-manager.md)
```

File: integrations/google-tag-manager.md [hunk 9]
```diff
@@ -3,6 +3,22 @@
 > **Notice of usage:** Google Tag Manager (GTM) and Google Analytics 4 (GA4) are third party services and we provide this integration as is.
 We support a set of custom events and Data Layer variables for use with GTM and GA4, however we have no control over what happens to them and how they are used. The basic setup examples we provide have been tested and verified to work with the Mews Booking Engine, however if you need a more complex setup then we cannot provide the support to do so and we recommend to ask a specialist to set it up and test it for you.
 
+
```
Comment (MikeAdamsMews) [inline integrations/google-tag-manager.md:6]: Copy/paste edited version here, from the integrations home page.

**Review by MikeAdamsMews — COMMENTED** [review]
Sounds good, just a few minor suggestions; and copy/paste so both instances are the same.

File: integrations/google-tag-manager.md (hunk 9 above)
Comment (progresak) [reply integrations/google-tag-manager.md:6]: Done, thanks for corrections 👍

**Review by MikeAdamsMews — APPROVED** [review]

**Pull request merged**